tests/install_upgrade_operators/console_cli_download/utils.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from kubernetes.dynamic import DynamicClient
+from timeout_sampler import TimeoutExpiredError, TimeoutSampler
+
+from utilities.constants import TIMEOUT_1MIN, TIMEOUT_5SEC, VIRTCTL_CLI_DOWNLOADS
+from utilities.infra import get_console_spec_links
+
+LOGGER = logging.getLogger(__name__)
+CUSTOMIZED_VIRT_DL = "customized-virt-dl"
+
+
+def validate_custom_cli_downloads_urls_updated(
+    admin_client: DynamicClient,
+    new_hostname: str | None = None,
+    original_virtctl_console_cli_downloads_spec_links: list[str] | None = None,
```

Comment:
why so big argument name?

I think something like "original_links" is clear enough